packages/camera/camera/example/lib/main.dart

```diff
@@ -100,15 +107,16 @@ class _CameraExampleHomeState extends State<CameraExampleHome>
   @override
   void didChangeAppLifecycleState(AppLifecycleState state) {
     // App state changed before we got the chance to initialize.
-    if (controller == null || !controller.value.isInitialized) {
+    if ((controller?.value.isInitialized ?? false)) {
```

Comment:
You've changed the meaning of this condition; it will now return early if `controller` is non-null and initialized, which is exactly the opposite of the intent.